test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -404,8 +404,8 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			step( 'Can enter post title and content', async function() {
 				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
 				await gEditorComponent.enterTitle( blogPostTitle );
-				await gEditorComponent.enterText( blogPostQuote );
-				return await gEditorComponent.ensureSaved();
+				return await gEditorComponent.enterText( blogPostQuote );
+				// return await gEditorComponent.ensureSaved();
```

Comment:
@bsessions85 What do you think about disabling `ensureSaved` in this step? The test works as expected and there are no failed runs.